packages/elasticsearch-plugin/src/elasticsearch.service.ts

```diff
@@ -193,13 +192,14 @@ export class ElasticsearchService implements OnModuleInit, OnModuleDestroy {
         );
         if (groupByProduct) {
             try {
-                const { body }: { body: SearchResponseBody<ProductIndexItem> } = await this.client.search({
-                    index: indexPrefix + PRODUCT_INDEX_NAME,
+                const { body }: { body: SearchResponseBody<VariantIndexItem> } = await this.client.search({
+                    index: indexPrefix + VARIANT_INDEX_NAME,
                     body: elasticSearchBody,
                 });
+                const totalItems = await this.totalHits(ctx, input, groupByProduct);
```

Comment:
totalItems now needs a separate search? Does this have any noticeable perf impact?

Reply:
Ok, got it. I'll see if I can test perf on this point. Although if that is the only way to do it, the results are moot.